DelegateFun/Sorter/Sorter.csproj

```
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>

```

```suggestion
    <TargetFramework>netstandard2.1</TargetFramework>
```
Need C# 8 to use nullable.